x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/scalar/whitelist/InternalEqlScriptUtils.java

```diff
@@ -36,4 +37,9 @@ public static Boolean startsWith(String s, String pattern) {
     public static String substring(String s, Number start, Number end) {
         return (String) SubstringFunctionProcessor.doProcess(s, start, end);
     }
+
+    public static Boolean stringContains(String string, String substring) {
```

Comment:
before `substring` for alphabetical order

Reply:
Moved.